jsonnet/kube-prometheus/main.libsonnet

```diff
@@ -19,6 +19,8 @@ local utils = import './lib/utils.libsonnet';
     common: {
       namespace: 'default',
       platform: null,
+      scrapeConfigReloader: true,
```

Comment:
I don't think this should be conditional since we thrive to monitor all the applications that we run.

Reply:
@dgrisonnet - understood, my thinking was that there might be situations where the downstream might be injecting a proxy for example or setting the reloader to listen on localhost where it wouldn't be scrape-able via the service monitor. Any concerns in that respect?